### ISSUE TITLE
training on  Imagenet data is not working

### ISSUE THREAD

**Issue by virtuoso108** [jameslahm/RepViT#75]
I am using one GPU so I ran the command like following.

python -m torch.distributed.launch --nproc_per_node=1 --master_port 12346 --use_env main.py --model repvit_m0_9 --data-path /media/iipl/DATA/DAT-main/imagenet --dist-eval

I trained for 300 epochs and the test acc1 is 0.094 and test acc5 is 1.338 which is very poor performance.
How can I train for classification on Imagenet with only one GPU?
Is there any learning rate and batch size for training with only one GPU?

I'd appreciate any help.
Thank you,

![repvit_train_not_Working_results_screenshot](https://github.com/user-attachments/assets/fbc33838-001c-42e8-82a4-d43caa5fa42a)
